modules/tests/src/test/scala/ch/epfl/scala/debugadapter/ScalaStackTraceTests.scala

```diff
@@ -0,0 +1,499 @@
+package ch.epfl.scala.debugadapter
+
+import ch.epfl.scala.debugadapter.testfmk.*
+
+class ScalaStackTraceTests extends ScalaDebugTests(ScalaVersion.`3.1+`) {
```

Comment:
Reduce number of tests. It is important to test:
- the filtered frames
- the order of frames